doc/help/elektra-spec.md

```diff
@@ -21,7 +21,7 @@ key database semantics (new plugins might be necessary, though).
 The most simple use is to enlist all keys that will be used by an application
 and maybe give a description for them (we use ini syntax in this document):
 
-```
+```ini
```

Comment:
The frontend uses [highlight.js](https://highlightjs.org/). A list of formats and color schemes can be found [here](https://highlightjs.org/static/demo/).